assets/i18n/id.json

```diff
@@ -25,245 +24,238 @@
-    "dgc-windows-wallet-modal-full-wallet": "Dogecoin Core, di sisi lain, adalah dompet \"full\". Sinkronisasinya dilakukan dengan cara mengunduhnya, menyediakan dompet Dogecoin yang berfungsi dengan baik.", 
+    "dgc-windows-wallet-modal-dogecoin-core-desc": "Dompet Dogecoin Yang \"Penuh\".", 
+    "dgc-windows-wallet-modal-dogecoin-core-guide": "Panduan Pengguna yang Dikelola Komunitas.", 
+    "dgc-windows-wallet-modal-getting-started": "Pelajari cara mengatur dompetmu dengan <a href=\"/getting-started\" target=\"_blank\">Panduan Pemula</a>.", 
+    "dgc-windows-wallet-modal-differences": "Apa perbedaan antara keduanya?", 
+    "dgc-windows-wallet-modal-sync": "Kamu harus melakukan sinkronisasi dengan blockchain sebelum menggunakan Dogecoin.", 
+    "dgc-windows-wallet-modal-ledger": "Ini berarti, Kamu memerlukan papan pencatatan Dogecoin (\"ledger\") untuk menggunakannya.", 
+    "dgc-windows-wallet-modal-light-wallet": "Multidoge adalah dompet versi \"ringan\" yang mensinkronkan dengan blockchain menggunakan proses \"pemindaian\", sehingga memungkinkan waktu penyelarasan relatif lebih singkat.",
+    "dgc-windows-wallet-modal-full-wallet": "Sementara, Dogecoin Core adalah dompet versi \"penuh\", dimana proses sinkronisasi dilakukan dengan mendownload keseluruhan data blockchain, sehingga mampu menyediakan dompet Dogecoin yang sungguh handal.", 
     "dgc-windows-wallet-modal-prosandcons": "Keuntungan dan Kerugian?", 
-    "dgc-windows-wallet-modal-different-strokes": "Masing - masing dompet memiliki keuntungan dan kerugiannya.", 
-    "dgc-windows-wallet-modal-multidoge-pro": "MultiDoge adalah sebuah light wallet, jadi itu hanya membutuhkan \"skim\" melalui blockchain sebelum dapat digunakan. Hal ini dilakukan dengan cepat.", 
-    "dgc-windows-wallet-modal-multidoge-con": "Tetapi dompet tersebut seharusnya hanya digunakan untuk penggunaan yang normal. Itu <strong>tidak dapat</strong> digunakan untuk menambang Dogecoin, karena dompet akan menjadi lambat dan tidak responsif jika digunakan untuk menambang Dogecoin.", 
-    "dgc-windows-wallet-modal-dogecoin-core-con": "Di sisi lain, Dogecoin Core mengunduh seluruh blockchain. Sinkronisasi awalnya jauh lebih lambat dibandingkan dengan MultiDoge, dan membutuhkan lebih banyak ruang.", 
-    "dgc-windows-wallet-modal-dogecoin-core-pro": "Tetapi ini adalah keuntungan karena aman untuk menambang Dogecoin dengan dompet ini, dompet ini dapat menangani transaksi dari pembayaran. Ini juga akan membantu menjaga jaringan Dogecoin.", 
+    "dgc-windows-wallet-modal-different-strokes": "Kedua dompet tersebut punya kelebihan dan kekurangan masing-masing.", 
+    "dgc-windows-wallet-modal-multidoge-pro": "MultiDoge itu dompet yang sangat ringan, jadi hanya \"memindai\" dalam blockchain sebelum menjadi tidak berguna lagi. Proses ini dilakukan sangat cepat.", 
```

Comment:
i think the translation here quite wrong compared to the original `en.json`

```
MultiDoge is a light wallet, so it only needs to \"skim\" through the blockchain before it becomes usable. This is done quickly
```

It should be
```
MultiDoge merupakan dompet yang ringan, hanya butuh untuk \"memindai\" pada blockchain sebelum dapat digunakan. Proses ini dilakukan sangat cepat.
```

Correct me if i'm wrong